projects/skcms/build.sh

```diff
@@ -30,4 +30,4 @@ cp iccprofile_seed_corpus.zip $OUT/iccprofile_info_seed_corpus.zip
 cp iccprofile_seed_corpus.zip $OUT/iccprofile_atf_seed_corpus.zip
 cp iccprofile_seed_corpus.zip $OUT/iccprofile_transform_seed_corpus.zip
 # They all use the same dictionary file.
-cp iccprofile.dict $OUT/iccprofile.dict
+cp iccprofile.dict $OUT/iccprofile.dict
```

Comment:
Try not to make unnecessary changes, it kind of clogs the git history/blame. Can you undo this please?